### DISCUSSION TITLE
"prismaSchemaFolder" doesn't exist

### DISCUSSION THREAD

**Discussion by preneyv-getCaas** [prisma/prisma#24672] [Q&A]
### Bug description

In my project I have this config for my prisma generator :  
`generator client {
  provider = "prisma-client-js"
  previewFeatures = ["prismaSchemaFolder"]
}`

But I have this error : 
`The preview feature "prismaSchemaFolder" is not known. Expected one of: deno, driverAdapters, fullTextIndex, fullTextSearch, metrics, multiSchema, postgresqlExtensions, tracing, views`

Can somebody tell me what happens ? I have the latest version of prisma and @prisma/client

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @preneyv-getCaas 👋 

Can you please share the version of Prisma you have installed? The `prismaSchemaFolder` preview feature was introduced in version 5.15.0. If you are using Visual Studio Code, after updating Prisma Client, you might need to restart the TypeScript server to recognize the new preview feature.

**Comment by LysenkovVA** [comment]
Same error in WebStorm 2023.2.5 on macos 14.5 (23F79). Prisma and prisma/client 5.16.1. Rebooting the system and IDE did not help

**Reply by LysenkovVA** [reply]
Everything seems to be working. The problem is with the unupdated Prisma ORM plugin from JetBrains

**Reply by TommyFreya** [reply]
Same error in WebStorm 2024.1.3 on macos 14.5.   Have you resolved the issue?

**Reply by preneyv** [reply]
Hi ! Thanks for your answer. No i didn't. I tried to re-install prisma orm plugin in webstorm, but no success.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
